### DISCUSSION TITLE
How to rename existing (plex created) collections?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#1030] [Q&A] [answered]
Hello,

I have dynamic collections set up like this, and its working good:

```
dynamic_collections:
  Trakt Liked Lists:
    type: trakt_liked_lists
    data: me
    remove_suffix: Collection, Collections, Filmreihe, Filmreihen
    remove_prefix: The, Die
  TMDb Collections:
    type: tmdb_collection
    remove_suffix: Collection, Collections, Filmreihe, Filmreihen
    remove_prefix: The, Die
```

Now, every time, I add some new movies, which have collections in TMDB, Plex creates a new collection.

Those collections have the suffix "Collection" or "Filmreihe".

Problem is, after I run PMM, I got collection doublettes. 

Those, Plex created (say "supermovie collection") and those, PMM generated (say "supermovie"). 
PMM stripped the suffix fine, but only for its generated collections.

How can I tell PMM to remove suffixes for all the Plex generated collections too?

_I found [this](https://github.com/meisnate12/Plex-Meta-Manager/discussions/711), but it seems to be depreciated._

Thank you.

**Comment by meisnate12** [comment, marked as answer]
if youre using TMDb dynamic_collections i would just turn off plex creating collections

**Reply by gitttt54** [reply]
Despite, not having an option for that, this is actually a very good idea! Thank you very much! I'll disable Collections now.

**Reply by gitttt54** [reply]
Appendix: 
I had removed all collections and I have set up Plex Libraries to not create collections and still (after a week of monitoring this and some restarts of the Plex Docker Container) plex creates collections for movies in the library to where I added a missing movie from a collection (which did not exist before).
Still no luck.
